test/server/typegen.ts

```diff
@@ -363,6 +363,11 @@ test('typegen', async () => {
         }
       }
     }
+
+    export type Tables<T extends keyof Database["public"]["Tables"]> =
```

Comment:
@soedirgo as mentioned, should we make this more generic? Otherwise we kind of need a helper for insert/updates also and for all different schemas? Wdyt?

Reply:
updated the PR